runtime/vara/src/weights/pallet_gear.rs

```diff
@@ -229,151 +229,162 @@ impl<T: frame_system::Config> pallet_gear::WeightInfo for SubstrateWeight<T> {
     /// The range of component `r` is `[0, 1]`.
     fn gr_reply(r: u32, ) -> Weight {
         // Proof Size summary in bytes:
-        //  Measured:  `0`
-        //  Estimated: `0`
-        // Minimum execution time: 99_615_000 picoseconds.
-        Weight::from_parts(106_812_220, 0)
-            // Standard Error: 92_523
-            .saturating_add(Weight::from_parts(18_397_180, 0).saturating_mul(r.into()))
+        //  Measured:  `0 + r * (141 ±0)`
+        //  Estimated: `2928`
+        // Minimum execution time: 65_333_000 picoseconds.
+        Weight::from_parts(67_988_414, 2928)
+            // Standard Error: 42_177
+            .saturating_add(Weight::from_parts(34_587_476, 0).saturating_mul(r.into()))
+            .saturating_add(T::DbWeight::get().reads((1_u64).saturating_mul(r.into())))
```

Comment:
TODO: remove read writes that depend on variadic param from all of the syscalls and instructions